bootstrapers/hychart/utils/useTokenChecker.ts

```diff
@@ -1,9 +1,19 @@
 import { auth } from "@bcrumbs.net/bc-api";
 import { useRouter } from "next/router";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 
-export function useTokenChecker() {
+export const useTokenChecker = () => {
   const router = useRouter();
+  const [hasToken, setHasToken] = useState(false);
+
+  const handleLogOut = () => {
+    try {
+      auth.clearAllAppStorage();
+      setHasToken(false);
+    } catch (error) {
+      console.error("Error logging out:", error);
+    }
+  };
```

Comment:
No, need to put this here, you can keep it in the header, using " useTokenChecker" in two places will duplicate running them without a reason